distrho/extra/String.hpp

```diff
@@ -658,7 +663,7 @@ class String
         uint i=0, j=0;
         uint charArray3[3], charArray4[4];
 
-        char strBuf[kTmpBufSize+1];
+        char* strBuf = (char*)malloc(kTmpBufSize + 1);
```

Comment:
does MSVC complain here too? a few lines above we can clearly see we limit `kTmpBufSize` to `65536`.
I sugest to make kTmpBufSize always 65536 for MSVC